src/components/dataset/query/sidebar/panels/CreateSnapshotPanel.jsx

```diff
@@ -23,40 +22,60 @@ const styles = theme => ({
   cancelButton: {
     alignSelf: 'flex-end',
   },
+  textField: {
+    backgroundColor: theme.palette.common.white,
+    borderRadius: '5px',
```

Comment:
can we use a theme.spacing for these?

Reply:
Yeah, we could use `theme.spacing(0.5)`, which would be 4px. This only applies to the white background, though. The `variant="outlined"` prop is what gives the text fields the grey border, which gets all its styling from default MUI classes, so that border radius would still be 5px.

Reply:
It might be useful to make a reusable Jade text field component with its own custom attributes? It would let us be consistent with Terra and we wouldn't have to constantly override MUI styling